community/CCLAs/CCLA Instructions.md

```diff
@@ -0,0 +1,8 @@
+## Instructions for handling Corporate Contributor License Agreements (CCLAs)
+
+* Step 1: Download the CCLA from [here]() 
+* Step 2: Complete the form
+* Step 3: Open a PR in the https://github.com/onnx/onnx repo with the completed CCLA attached in the [CCLA directory](https://github.com/onnx/onnx/tree/master/community/CCLAs)
```

Comment:
I suggest we keep this out of the main ONNX repo and submit to the steering committee repo instead. People developing on/with ONNX don't need to download random company's CCLA submissions.